### DISCUSSION TITLE
Ban for too many closed PRs

### DISCUSSION THREAD

**Discussion by yangwao** [kodadot/nft-gallery#8407] [General]
I'm considering setting a ban for creating too many closed pull requests vs merged ratio. Let's say hitting ratio 5:1 

Cases like this eat the precious time of our small team. Once team would be bigger or has room to answer low-quality PRs, we can lift this ban. 

Thanks for all your contributions, **yet our time is precious, and as we've decided to do development in public, we can't afford to invest time in closed PRs.**

Important figure is amount of comments when with good will our team trying leg you up, but if your code quality and contribution doesn't progress to better, better go to train on your own or pay someone 👌😁

https://github.com/kodadot/nft-gallery/pulls?q=is%3Apr+sort%3Aupdated-desc+author%3AAshutoshSingh00001+is%3Aclosed

![image](https://github.com/kodadot/nft-gallery/assets/5887929/62e64157-ba5e-40f4-bbf8-3b6941241111)

https://github.com/kodadot/nft-gallery/pulls?q=is%3Apr+sort%3Aupdated-desc+author%3AAshutoshSingh72+is%3Aclosed

![image](https://github.com/kodadot/nft-gallery/assets/5887929/417b9a45-ad74-4750-8cf3-d334e0539ae1)

**Comment by roiLeo** [comment]
🙏 I don't want to be unpleasant, but it's true that after a while, it's tiring.

**Reply by yangwao** [reply]
I was about to enforce it on 4/11 when I saw that one PR closed, but something else has crossed my attention and didn't return, sry being late, but rather now than never.

If anything else like this keeps occurring, please report it for investigation

**Comment by prury** [comment]
i support the ban system

**Comment by exezbcz** [comment]
helo sir @pbkompasz, just letting you know something like this exist 

![telegram-cloud-photo-size-1-5093797884566023320-y](https://github.com/kodadot/nft-gallery/assets/90852205/ebac923b-921d-429e-80c1-cd53c100c915)

**Reply by pbkompasz** [reply]
I know and I want to apologize for it. Those PRs were from last year, when I started to contribute to the project and due to various reasons I couldn't merge some them. Now I am starting to contribute again and I will make sure to create only useful PRs.
